processes/dags/pubmed_migration.py

```diff
@@ -231,15 +244,16 @@ def extract_pubmed_data() -> None:
     """Extracts case-reports from pubmed data and stores result on S3
     """
     pattern = "*.xml.tar.gz"
+    #pattern = "comm_use.A-B.xml.tar.gz"
     ftp_path = '/pub/pmc/oa_bulk'
     root_dir = '/usr/local/airflow'
     temp_dir = os.path.join(root_dir, 'temp')
     bucket_name = 'supreme-acrobat-data'
-    prefix = 'case_reports/pubmed/original'
+    prefix = 'test/pubmed/original'
```

Comment:
Don't we need to change our paths back?

Reply:
I'm still testing for utils file. will update soon